build-system/tasks/prepend-global/index.js

```diff
@@ -169,10 +169,15 @@ function enableLocalDev(config, target, configJson) {
   }
   const TESTING_HOST = process.env.AMP_TESTING_HOST;
   if (typeof TESTING_HOST == 'string') {
+    const TESTING_HOST_FULL_URL = TESTING_HOST.match(/^[a-zA-Z]+:\/\//) ?
```

Comment:
Since the protocol is expected to be just one of `http://` or `https://`, we can use the following regex:
```
/^https?:\/\//
```

Reply:
+1